tests/functional/object/mcg/test_loadbalancer_subnet_config.py

```diff
@@ -0,0 +1,163 @@
+import json
+import logging
+from time import sleep
+
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import (
+    aws_platform_required,
+    polarion_id,
+    tier2,
+    mcg,
+    red_squad,
+)
+from ocs_ci.framework.testlib import MCGTest
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+@mcg
+@red_squad
+@aws_platform_required
+class TestLBSubnetConfig(MCGTest):
+
+    @tier2
+    @polarion_id("OCS-4716")
+    def test_lb_subnet_config(self, awscli_pod):
+        """
+        Test whether MCG's load balancer subnet config allows connections
+        to the load balancer only from the specified subnets:
+
+        1. Test local connection to the load balancer
+        2. Test connectivity from a pod to the load balancer
+        3. Patch the local IP as the only subnet in noobaa/noobaa.spec.loadBalancerSourceSubnets
+        4. Verify that the pod is no longer able to connect to the load balancer
+        5. Verify that local connection to the load balancer is still available
+        6. Revert the patch
+        7. Verify that the pod is able to connect to the load balancer
+        8. Verify that local connection to the load balancer is still available
+
+        This test uses the awscli pod as a second subnet to test, but it can be
+        any pod that can connect to the load balancer.
+        """
+        noobaa_obj = OCP(
+            kind="noobaa",
+            namespace=config.ENV_DATA["cluster_namespace"],
+        )
+        local_ip = exec_cmd("curl -s https://ifconfig.me").stdout.decode().split(" ")[0]
+        lb_host_name = OCP(
+            kind="service",
+            namespace=config.ENV_DATA["cluster_namespace"],
+            resource_name="s3",
+        ).get()["status"]["loadBalancer"]["ingress"][0]["hostname"]
+        port = 443  # This is the default port for the load balancer
+
+        # 1. Test local connection to the load balancer
+        local_can_connect = self.test_connectivity(lb_host_name, port, pod_obj=None)
```

Comment:
```suggestion
        local_can_connect = self.test_connectivity(lb_host_name, port)
```

Reply:
Fixed in https://github.com/red-hat-storage/ocs-ci/pull/13173/commits/7fc0aa3f04248b7d443e6e21631902cde59c91d1